### DISCUSSION TITLE
2021-05-12 Weekly Meeting 3 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#27] [Weekly Meetings]
Hello everyone! Thank you to those that came out for our OKR workshop last week! It was totally worth waking up at 5:00 AM PDT for :sweat_smile: 

If you missed the last meeting, we came up with these objectives for OMI:

1. We collaborate to make the metaverse more human.

2. We empower the people making the metaverse.

3. We are creating a cooperative, sustainable culture of innovation.

4. We incrementally deliver value, with intent.

5. We must achieve resiliency through diversity, and freedom of choice.

![image](https://user-images.githubusercontent.com/1753624/117747633-6075bd00-b1c3-11eb-9e3e-56dc25c4b682.png)

This meeting wasn't recorded, but you can check out the full FigJam project [here](https://www.figma.com/file/HcSUQRYUHjTcVyGIA0fu1u/OMi-OKR-Workshop-Part-II?node-id=0%3A1).

## Agenda

Weekly Meeting 3 is happening on Wednesday 5/12/2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

We will be finishing our OKRs by defining key metrics for each of the 5 objectives. After that we'll go into our scope of work and discussing our proposal process.

After this meeting is the part where people get to start to start submitting their ideas for interoperability protocols and we get to figure out how we can start working together. For a lot of you, this next part is the part you've been waiting for and we thank you for your patience!

If you have other items that you'd like to add to the agenda please comment below and we'll get to everything we can. Give things a :+1: if you'd also like to see them on the agenda so we know what to prioritize.

Hope to see you all there!

**Comment by mrmetaverse** [comment]
Clarity around what we are looking for from the group for OMI W3C Chair nominations. What are we looking for, and what are the specifications of a Chair? (per our discussion)

**Comment by mrmetaverse** [comment]
Let's start with announcements and other agenda items before getting into KR's 

- W3C Chairs
- proposal process 
- Jin mentioned a NFT bot :) 
- when can we find time to have an open discussion about incentives

Then we will dive into KR's for the remaining time.

**Reply by mrmetaverse** [reply]
Notes captured from your explanation today @robertlong : 

**Our draft Proposal Process:** Create a github discussion, and write a draft explainer (use accessible language as best as you can) of your proposal solution, or draft specification. These need to have more than one implementer to be considered for an official specification. We can formalize our process as time goes, but the primary goal is to get folks comfortable submitting and discussing ideas on Github discussions, and ensure experiments are adopted. 

We can always refine the process as we go.

**Comment by mrmetaverse** [comment]
Figjam for session III: https://www.figma.com/file/6F0kdsO2o4D97RT74m7KkK/OMi-OKR-Workshop-Part-III?node-id=0%3A1
